recipes/mender-client/debian-master/rules.legacy.patch

```diff
@@ -0,0 +1,12 @@
+--- rules	2023-12-13 09:06:16.575373793 +0100
++++ rules.legacy	2023-12-13 09:07:10.555779873 +0100
+@@ -6,7 +6,8 @@
+ override_dh_auto_configure:
+ 	dh_auto_configure -- \
+ 		-D CMAKE_BUILD_TYPE=Release \
+-		-D BOOST_ALL_DYN_LINK=ON
++		-D BOOST_ALL_DYN_LINK=ON \
```

Comment:
I suspect this might conflict with the `MENDER_DOWNLOAD_BOOST` flag, since it internally uses static linking (it has to, because there are no system libraries to link to).

Reply:
Do you mean we should should then replace `BOOST_ALL_DYN_LINK` with `MENDER_DOWNLOAD_BOOST` in the legacy builds?

Reply:
CI seems to be happy. If there is a conflict, do you expect it to shown at build time or at run time?

Reply:
Yes. Possibly also for the normal builds. I removed it from the cmake files because it conflicted with the `MENDER_DOWNLOAD_BOOST` flag, and have yet to see any problems with it.